client/src/components/DescriptionInput.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+const TextInput = styled.textarea`
+  height: 150px;
+  width: 250px;
+  margin-top: 10px;
+  border-radius: 10px;
+  border: none;
+  padding-left: 10px;
+  padding-top: 5px;
+  padding-right: 10px;
```

Comment:
I'd go with the clockwise format if you have more than two values for padding or margin. 
`padding: 5px 10px 0px 10px ;`

Reply:
that's a good idea, thanks!